// gulp-named/dist/gulp-named.common.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var gulp = _interopDefault(require('gulp'));

var task = function () {
  var args = [], len = arguments.length;
  while ( len-- ) args[ len ] = arguments[ len ];

  return gulp.task.apply(gulp, args)
};

var src = function () {
  var args = [], len = arguments.length;
  while ( len-- ) args[ len ] = arguments[ len ];

  return gulp.src.apply(gulp, args)
};

var dest = function () {
  var args = [], len = arguments.length;
  while ( len-- ) args[ len ] = arguments[ len ];

  return gulp.dest.apply(gulp, args)
};

var watch = function () {
  var args = [], len = arguments.length;
  while ( len-- ) args[ len ] = arguments[ len ];

  return gulp.watch.apply(gulp, args)
};

var input = src;
var output = dest;

exports.task = task;
exports.src = src;
exports.dest = dest;
exports.watch = watch;
exports.input = input;
exports.output = output;
